node-backend/subProblemHandler.js

```diff
@@ -99,7 +99,7 @@ module.exports = function(db) {
   }
 
   function update(request, response, next) {
-    logger.debug('UPDATE /workspaces/:id/problems/:subProblemId');
+    logger.debug('Update subproblem: '+ request.params.subProblemId);
```

Comment:
the new log line actually gives less information.

Reply:
Will print the rest of params too. But I think logger actually doesn't know how to substitute :id etc so less info was printed before the change :)

Reply:
yes, but you understood what I meant anyway :P